packages/config/src/constants.js

```diff
@@ -0,0 +1,114 @@
+const SINGLE_CELL_WITH_HEATMAP_VIEWS = {
+  obsSets: [4, 0, 4, 4],
+  obsSetSizes: [8, 0, 4, 4],
+  scatterplot: [0, 0, 4, 4],
+  heatmap: [0, 4, 8, 4],
+  featureList: [8, 4, 4, 4],
+};
+
+const SINGLE_CELL_WITHOUT_HEATMAP_VIEWS = {
+  obsSets: [10, 6, 2, 6],
+  obsSetSizes: [8, 1, 4, 6],
+  scatterplot: [0, 0, 8, 12],
+  featureList: [8, 6, 2, 6],
+};
+
+const SPATIAL_TRANSCRIPTOMICS_VIEWS = {
+  scatterplot: [0, 0, 3, 4],
+  spatial: [3, 0, 5, 4],
+  obsSets: [8, 0, 4, 2],
+  featureList: [8, 0, 4, 2],
+  heatmap: [0, 4, 6, 4],
+  obsSetFeatureValueDistribution: [6, 4, 6, 4],
+};
+
+const SPATIAL_TRANSCRIPTOMICS_WITH_HSITOLOGY_VIEWS = {
+  spatial: [0, 0, 6, 6],
+  heatmap: [0, 6, 8, 6],
+  layerController: [8, 6, 4, 6],
+  obsSets: [9, 0, 3, 6],
+  featureList: [6, 0, 3, 6],
+};
+
+const IMAGE_VIEWS = {
+  spatial: [0, 0, 8, 12],
+  layerController: [8, 0, 4, 7],
+  description: [8, 9, 4, 5],
+};
+
+export const NO_HINTS_CONFIG = {
+  title: "Don't use any hints",
+  views: {},
+  coordinationValues: {},
+};
+
+export const HINTS_CONFIG = {
+  E: {
```

Comment:
What do the keys of this object represent (E, B, C, D)? And the sub-objects (6, 5, 4, 3, 2, 1; 2, 1; 2, 1; 1)? If nothing, can we use plain arrays instead?

Reply:
We can use plain arrays. The reason I implemented it this way was because I initially thought that I will save iterations over arrays with having an object with unique keys. Now I see that this approach makes the code more complicated than it should be. I have refactored the structure of HINTS_CONFIG. Can you take a look?